docker-app/qfieldcloud/core/models.py

```diff
@@ -1809,16 +1810,25 @@ def save(self, recompute_storage=False, *args, **kwargs):
                     file_storage_bytes=Sum("versions__size", default=0)
                 )["file_storage_bytes"]
 
+            additional_update_fields.add("file_storage_bytes")
+
         # Ensure that the Project's storage_keep_versions is at least 1, and reflects the plan's default storage_keep_versions value.
         if not self.storage_keep_versions:
             self.storage_keep_versions = (
                 self.owner.useraccount.current_subscription.plan.storage_keep_versions
             )
 
+            additional_update_fields.add("storage_keep_versions")
+
         assert self.storage_keep_versions >= 1, (
             "If 0, storage_keep_versions mean that all file versions are deleted!"
         )
 
+        if "update_fields" in kwargs and kwargs["update_fields"] is not None:
```

Comment:
```suggestion
        if kwargs.get("update_fields") is not None:
```

Do I miss a scenario where the above suggestion would not work? Most of the existing codebase uses this pattern to check if keys exists.

No strong opinion shared here.